integreat_cms/cms/utils/zammad.py

```diff
@@ -0,0 +1,243 @@
+"""
+Zammad API helper functions
+"""
+
+from abc import abstractmethod
+from typing import TYPE_CHECKING
+
+import requests
+from django.conf import settings
+from django.core.cache import cache
+
+if TYPE_CHECKING:
+    from ..models import Region
+
+
+class ZammadAPI:
+    """
+    Zammad API Wrapper. This is intended to be used as a UserChat parent class.
+    """
+
+    @property
+    @abstractmethod
+    def zammad_id(self) -> int:
+        """
+        Property that has to be defined in child classes
+        """
+        raise NotImplementedError
+
+    @property
+    @abstractmethod
+    def region(self) -> "Region":
```

Comment:
```suggestion
    def region(self) -> Region:
```

or am I missing something? 😅

Reply:
As Region is only imported during type checking, `Region` would be undefined. We cannot import Region due to cyclic imports.